src/main/java/com/endava/utility/Main.java

```diff
@@ -0,0 +1,24 @@
+package com.endava.utility;
+
+import com.endava.utility.list.Lists;
+import java.util.List;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        List<String> listOfStrings = Lists.createList("jedan","dva","tri");
+        List<Integer> listOfIntegers = Lists.createList(1, 2, 3);
+        printList(listOfStrings);
+        printList(listOfIntegers);
+
+        List<String> listOfStrings2 = com.google.common.collect.Lists.newArrayList("jedan","dva","tri");
```

Comment:
how can you import newArrayList function? 
It's hard to read when you have complete path here.

Reply:
Right. It could be imported with: import static com.google.common.collect.Lists.newArrayList;


Because I created my own "Lists" implementation, there could be only one qualified import. I have removed that as well :).